app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java

```diff
@@ -20,7 +20,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_welcome);
 
-        getSupportActionBar().hide();
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().hide();
```

Comment:
We no longer need the null check as the app now uses the `NoActionBar` theme. `getSupportActionBar().hide();` can be completely removed as the activity has no action bar. Other activities have a custom action bar.